core/common/src/main/java/com/google/jetpackcamera/core/common/CommonModule.kt

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I have mixed thoughts about this, since DI is just the method of providing dependencies, my feeling is it's best to keep those dependencies organized.

Reply:
Fine with me 👍